src/main/java/duke/entities/Deadline.java

```diff
@@ -0,0 +1,18 @@
+package duke.entities;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d/M/[uuuu][uu] HHmm");
+    private static final DateTimeFormatter printFormatter = DateTimeFormatter.ofPattern(("d/M/yyyy HHmm"));

```

Comment:
>  Constant names must be all uppercase using underscore to separate words.

So perhaps they could be named `DATE_TIME_FORMATTER` and `PRINT_DATE_TIME_FORMATTER`